tests/unit/Alteration.spec.ts

```diff
@@ -211,7 +211,7 @@ describe('Alteration component', () => {
               roles: [
                 {
                   roleClass: 'OFFICER',
-                  roleType: 'CEO'
+                  roleType: 'Chief Executive Officer'
```

Comment:
Are you sure this should be the string value and not the key?

Reply:
Sample data from API:

<img width="851" height="232" alt="image" src="https://github.com/user-attachments/assets/8ae779e6-28cf-41b2-a720-0f6ecf323fee" />